### ISSUE TITLE
Confusing/not adequate verbosity in ARIA-enabled menus

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#773]
**Reported by kennpetri on 2010-07-21 20:40**
When navigating ARIA-enabled menu widgets, NVDA properly announces that the cursor is on a menu and properly announces the number of menu items and position within the bar. When enter is pressed to enter the sub-menu/menu items, NVDA announces "menu [of [numbers](number])" but in navigating via arrow key does not announce the name of the menu item. That is, we hear "2 of 3" but not the actual name of/label for the menu item itself. Also, occasionally, NVDA pulls in extraneous information. For example, that the sub-menu was implemented in a "table."

By contrast, JAWS 11 verbosity is more helpful. When entering the menu bar, JAWS says "Menu bar. To navigate, press right/down and left/up arrow keys." JAWS also accurately reads the menu item labels/names and uses an audio tone to indicate the user is "in" the menu.

Note: testing done using Firefox 3.6 

Please see http://archive.dojotoolkit.org/nightly/dojotoolkit/dijit/tests/test_Menu.html for a model implementation.

**Comment by leonardder** [comment]
I'm not sure whether I can still reproduce this. Many things have been changed since this was reported 7 years ago.

**Comment by derekriemer** [comment]
Closing as worksfome, because it seems that this is fixed after all these years. Please reopen if it isn't so.
